index.html

```diff
@@ -0,0 +1,845 @@
+          <p>We might never know what happened before inflation, at the very
+            beginning, because inflation erases everything that came before
+            it. All the chaos and randomness of the primordial moment are
+            swept away, forever out of our view.</p>
+          <p>“If you trace your cosmic roots,” said Abraham Loeb, a
+            Harvard-Smithsonian astronomer who was not part of the team, “you
+            wind up at inflation.”</p>
+          <div>
+            <div class="article-footer text-gray">
+              A version of this article appears in print on March 18, 2014,
+              Section A, Page 1 of the New York edition with the headline:
+              Space Ripples Reveal Big Bang’s Smoking Gun. <a href="http://www.nytreprints.com/">Order Reprints </a> |
+              <a href="https://www.nytimes.com/section/todayspaper"> Today’s
+                Paper</a> | <a
+                href="https://www.nytimes.com/subscriptions/Multiproduct/lp8HYKU.html?campaignId=48JQY">Subscribe</a>
+            </div>
+          </div>
+        </div>
+        <div class="clear"></div>
+        <div class="col-60 push-20">
+          <div class="col-50">
+            <button class="col-90 bg-primary">
+              Read 615 Comments
+            </button>
+          </div>
+          <div class="col-50">
+            <span class="wrap-social-media right">
+              <a href="#">
+                <i class="fa fa-facebook"></i>
+              </a>
+              <a href="#">
+                <i class="fa fa-twitter"></i>
+              </a>
+              <a href="#">
+                <i class="fa fa-envelope"></i>
+              </a>
+              <a href="#">
+                <i class="fa fa-share"></i>
+              </a>
+            </span>
+          </div>
+        </div>
+        <div class="clear"></div>
+        <div class="col-70">
+          <h4 class="title-bar">
+            <a href="https://www.nytimes.com/section/science/space?action=click&module=MoreInSection&pgtype=Article&region=Footer&contentCollection=Space%20and%20Astronomy">
+              More in Space and Astronomy
+            </a>
+          </h4>
+          <div class="more-article-box">
+            <a href="https://www.nytimes.com/2020/01/30/science/nasa-spitzer-space-telescope.html?action=click&module=MoreIn&pgtype=Article&region=Footer&action=click&module=MoreInSection&pgtype=Article&region=Footer&contentCollection=Space%20and%20Astronomy">
+            <div class="image">
+              <img src="https://static01.nyt.com/images/2020/02/04/science/30SCI-SPITZER1c/30SCI-SPITZER1c-threeByTwoSmallAt2X-v2.jpg" alt="image">
+              </div>
+              <div class="text-right platform">NASA/JPL</div>
+              <h5 class="no-padding no-margin">NASA’s Spitzer Space Telescope Ends 16-Year Mission of Discovery</h5>
+              <div class="text-gray timestamp">Jan. 30</div>
```

Comment:
Only content inside tag h5 should be link
Example: <a href="https://www.....">  <h5 class="no-padding no-margin">NASA’s Spitzer Space Telescope Ends 16-Year Mission of Discovery</h5></a>